### PR TITLE
feat(assets): Add temp cache busting on unversioned assets

### PR DESCRIPTION
**Pull request by evanpurkhiser**
After a recent CDN issue where our unversioned assets were not being
revalidated, we're adding this temporary cache busting locked to the
backend version.

We'll have to implement this differently for frontend only builds

### CODE REVIEW

File: src/sentry/utils/assets.py [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+
+      {% unversioned_asset_url 'sentry' 'sentry.css' cache_bust=True %}
+      =>  "/_static/dist/sentry/sentry.css?bust=xxx"
     """
+    args = (settings.STATIC_UNVERSIONED_URL.rstrip("/"), module, key.lstrip("/"))
 
-    return "{}/{}/{}".format(settings.STATIC_UNVERSIONED_URL.rstrip("/"), module, key.lstrip("/"))
+    if not cache_bust:
+        return "{}/{}/{}".format(*args)
+    return "{}/{}/{}?bust={}".format(*args, sentry.__build__)
```
Comment (billyvg) [inline src/sentry/utils/assets.py:28]: what about `v` or `ver`

```suggestion
    return "{}/{}/{}?ver={}".format(*args, sentry.__build__)
```

Comment (EvanPurkhiser) [reply src/sentry/utils/assets.py]: 👍

File: src/sentry/utils/assets.py [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+    we're busting caches when version_bust is True using a query parameter with
+    the currently deployed backend SHA. This will have to change in the future
+    for frontend only deploys.
+
     Example:
       {% unversioned_asset_url 'sentry' 'sentry.css' %}
       =>  "/_static/dist/sentry/sentry.css"
+
+      {% unversioned_asset_url 'sentry' 'sentry.css' cache_bust=True %}
+      =>  "/_static/dist/sentry/sentry.css?bust=xxx"
```
Comment (billyvg) [inline src/sentry/utils/assets.py:20]:
```suggestion
      =>  "/_static/dist/sentry/sentry.css?v=xxx"
```

**Review by billyvg — APPROVED** [review]

File: src/sentry/utils/assets.py (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/utils/assets.py:20]: damn

Comment (EvanPurkhiser) [reply src/sentry/utils/assets.py:20]: gonna follow up, I just wanna merge it

Comment (EvanPurkhiser) [reply src/sentry/utils/assets.py:20]: nvm the acceptance was canceled lol, I'll let it through

**Pull request merged**